features/old/switches/branding.feature

```diff
@@ -10,11 +10,11 @@ Feature: Branding switch
   Scenario: Dns link invites to upgrade
     Given provider "foo.3scale.localhost" has "branding" switch denied
     And I go to the dns settings page
-    And I follow "Change"
+    And I follow "Change" within ".string:has(#account_subdomain)"
```

Comment:
We can add a new entry to `selectors.rb` or even better a title to the change link. If the link had a title i.e. `<a title="Change account subdomain" ... >`, the step would be more robust:
```
```suggestion
    And I follow "Change account subdomain"
```

This dns/settings form is not yet patternfly and the change button will be moved out of the input eventually, rendering `within ".string:has(#account_subdomain)"` invalid.

Reply:
Makes sense, done